app/views/ingredients/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Ingredients</h1>
+
+<% @ingredients.each do |ingredient| %>
+  <p><%= ingredient.name %></p>
+  <p><%= ingredient.cost %></p>
+<% end %>
```

Comment:
This is great. Add some descriptive text so we know what the name and number showing up on the page represent. Practicing this habit now while these assessments are smaller will allow it to become muscle memory by the time you get to the final.